core/src/main/java/org/apache/carbondata/core/keygenerator/directdictionary/timestamp/TimeStampGranularityConstants.java

```diff
@@ -27,6 +27,10 @@
    * customized the start of position. for example "January 1, 2000"
    */
   String CARBON_CUTOFF_TIMESTAMP = "carbon.cutOffTimestamp";
+  /**
+   * default value of cutoff timestamp
+   */
+  String CARBON_CUTOFF_TIMESTAMP_DEFAULT_VAL = "1970-01-01 05:30:00";
```

Comment:
FYI:

This is set based on `configuration-parameters.md`.

Dictionary value of timestamp will be generated as following:
1. Convert the value to Date（d1）；
2. Get the ms of the Date value （ts = d1.getTime()）；
3. Calculate the key for ts. If ts < cutOffTimeStamp, then it be treated as bad record, otherwise carbondata will generate the key...